app/models/invoice_item.rb

```diff
@@ -6,5 +6,9 @@ class InvoiceItem < ApplicationRecord
   validates :unit_price, presence: true, numericality: true
   validates :status, presence: true
   
-  enum status: {"pending" => 0, "packaged" => 1, "shipped" => 2}
+  enum status: {"Pending" => 0, "Packaged" => 1, "Shipped" => 2}
+
+  def format_price_sold
+    (self.unit_price.to_f / 100).to_fs(:currency)
+  end
```

Comment:
See other comment, but this leads me to wonder how deep the rabbit hole goes, such as formatting created_at to .strftime("%_m/%d/%Y"), is this also a violation of MVC?